packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx

```diff
@@ -105,4 +108,34 @@ export const Examples = () => {
   );
 };
 
+export const Basic: ComponentStory<typeof PanelChrome> = (args: PanelChromeProps) => <PanelChrome {...args} />;
+
+const LoadingIcon = <PanelChrome.LoadingIndicator loading onCancel={() => {}} />;
+const ErrorIcon = <PanelChrome.ErrorIndicator error="Error text" onClick={() => {}} />;
+
+const leftItems = { LoadingIcon, ErrorIcon };
+
+Basic.argTypes = {
+  leftItems: {
+    options: Object.keys(leftItems),
+    mapping: leftItems,
+    control: {
+      type: 'select',
+      labels: {
+        LoadingIcon: 'With loading icon',
+        ErrorIcon: 'With error icon',
+      },
+    },
+  },
+};
+
+Basic.args = {
+  width: 400,
+  height: 200,
+  title: 'Title text',
+  children: (width: number, height: number) => (
+    <div style={{ height, width, background: '#22252b' }}>Description text</div>
```

Comment:
@JoaoSilvaGrafana thanks for the feedback, I managed to avoid creating extra renderer function for Basic story, please let me know what you think about it.

I have also adjusted onClick/onCancel handlers of Loading/Error indicators to use addon-actions.